### PR TITLE
ci: migrate devtools tests to GHA 

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -55,3 +55,21 @@ jobs:
...
+      - name: Initialize environment
+        uses: angular/dev-infra/github-actions/npm/checkout-and-setup-node@222de0bc054a6d2231243f5ea5ea97991a6a722a
+        with:
+          cache-node-modules: true
+      - name: Setup Bazel
+        uses: angular/dev-infra/github-actions/bazel/setup@222de0bc054a6d2231243f5ea5ea97991a6a722a
+      - name: Setup Bazel RBE
+        uses: angular/dev-infra/github-actions/bazel/setup-bazel-remote-exec@222de0bc054a6d2231243f5ea5ea97991a6a722a
+      - name: Install node modules
+        run: yarn install --immutable
```
Comment (devversion) [inline .github/workflows/ci.yml:71]: We are not using Yarn 2.x here

Comment (devversion) [reply .github/workflows/ci.yml:71]: Also need to change above I think (or didn't you have a PR for that?)

**Review by devversion — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (josephperrott) [reply .github/workflows/ci.yml:71]: I updated the change in components to using `--frozen-lockfile`, but had already created this one and had forgotten to update it as well.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 5a5bcd2565b8569abd16e8f42d05ed6bea1e89a7.

**Pull request closed without merging**